lib/app.js

```diff
@@ -200,13 +200,18 @@ function App () {
 
   initSSE(app)
 
+  // Data integration experimental endpoints created by James McLaughlin
```

Comment:
We can probably ditch these now.

Reply:
We can leave them as experimental, if you think we might want to resurrect this code at some point.  I don't recall exactly what they did.